test/money.html

```diff
@@ -46,58 +47,70 @@
 					}
 				}
 			},
-			localizedOutput = localizedOutputs[lang],
-			locAmount = localizedOutput[amount],
 			isEdge = !!window.StyleMedia,
 			ua = window.navigator.userAgent,
 			isSafari = ua.indexOf('Chrome') === -1 && ua.indexOf('Safari') !== -1,
 			alt = isEdge || isSafari;
 
-		test('amountEquals ', () => {
-			assert.equal(amountEquals(null, null), false);
-			assert.equal(amountEquals(null, false), false);
-			assert.equal(amountEquals(100, 100), false);
-			assert.equal(amountEquals({ amount }, { amount }), false);
-			assert.equal(amountEquals({ amount }, { amount, currency: 'USD' }), false);
-			assert.equal(amountEquals({ amount: null, currency: 'USD' }, { amount: null, currency: 'USD' }), false);
-			assert.equal(amountEquals({ amount, currency: null }, { amount, currency: null }), false);
-			assert.equal(amountEquals({ amount, currency: 'SEK' }, { amount }), false);
-			assert.equal(amountEquals({ amount, currency: 'EUR' }, { amount: -amount, currency: 'EUR' }), false);
-			assert.equal(amountEquals({ amount, currency: 'EUR' }, { amount, currency: 'EUR' }), true);
-			assert.equal(amountEquals({ amount: 1.01, currency: 'EUR' }, { amount: 1.009, currency: 'EUR' }, 2), true);
-			assert.equal(amountEquals({ amount: 1.01, currency: 'EUR' }, { amount: 1.009, currency: 'EUR' }, 3), false);
-			assert.equal(amountEquals({ amount: 1.100, currency: 'EUR' }, { amount: 1.1001, currency: 'EUR' }, 2), true);
-			assert.equal(amountEquals({ amount: -1.1001, currency: 'EUR' }, { amount: -1.100, currency: 'EUR' }, 2), true);
-			assert.equal(amountEquals({ amount, currency: 'EUR' }, { amount, currency: 'EUR' }, 4), true);
-		});
+		Object.keys(localizedOutputs).forEach(locale => {
+			const
+				localizedOutput = localizedOutputs[locale],
+				locAmount = localizedOutput[amount];
 
-		test('isAmount ', () => {
-			assert.equal(isAmount(null), false);
-			assert.equal(isAmount(false), false);
-			assert.equal(isAmount(100), false);
-			assert.equal(isAmount({ amount }), false);
-			assert.equal(isAmount({ currency: 'EUR' }), false);
-			assert.equal(isAmount({ amount, currency: 'EUR' }), true);
-			assert.equal(isAmount({ amount: amount.toString(), currency: 'EUR' }), false);
-			assert.equal(isAmount({ amount: null, currency: 'USD' }), false);
-			assert.equal(isAmount({ amount, currency: null }), false);
-		});
+			test('amountEquals ', () => {
```

Comment:
This test does not depend on locAmount, so you should take it out of the forEach.

Reply:
Moved it.